src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1176,32 +1176,7 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
             return receivedRequestVersion;
         }
         #endregion
-
-        #region SSL Version tests
-        [Theory]
-        [InlineData("SSLv2", HttpTestServers.SSLv2RemoteServer)]
-        [InlineData("SSLv3", HttpTestServers.SSLv3RemoteServer)]
-        public async Task GetAsync_UnsupportedSSLVersion_Throws(string name, string url)
-        {
-            using (HttpClient client = new HttpClient())
-            {
-                await Assert.ThrowsAsync<HttpRequestException>(() => client.GetAsync(url));
-            }
-        }
-
-        [Theory]
-        [InlineData("TLSv1.0", HttpTestServers.TLSv10RemoteServer)]
-        [InlineData("TLSv1.1", HttpTestServers.TLSv11RemoteServer)]
-        [InlineData("TLSv1.2", HttpTestServers.TLSv12RemoteServer)]
-        public async Task GetAsync_SupportedSSLVersion_Succeeds(string name, string url)
```

Comment:
The nice thing about this test is that it uses a different process to understand the TLS version.  If we, for example, messed up the enum alignment and had TLS 1.1 and 1.2 backwards our local test would pass (Open "1.2" (1.1 in reality), verify the connection reports "1.2" (1.1 in reality)).

So, I'd possibly leave these as OuterLoop, just to get that extra bit of validation.